public/data/HAT52.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "Hamza Ali",
+    "type": "personal",
+    "bio": "Tech Enthusiast!",
+    "avatar": "https://github.com/HAT52.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/HAT52",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Hami_Ali98",
+        "icon": "twitter"
+      },
+      {
+        "name": "Let's connect on LinkedIn",
+        "url": "linkedin.com/in/hamza-ali-8b79171aa",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing",
+        "date": "March 2020",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing"
+      },
+      {
+        "title": "Landed a job",
+        "date": "April 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Landed a job"
+      }
+    ]
+  }
```

I think there should be an endline here 😅 This can cause some problems with IDEs and such if there's no endline. 😅
```suggestion
}

```